### DISCUSSION TITLE
How to install on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by mozgbrasil** [ImageMagick/ImageMagick#4286] [Help]
Accessed

https://imagemagick.org/script/download.php

I downloaded the file

https://download.imagemagick.org/ImageMagick/download/binaries/magick

and I have the following return

```bash
marcio@marcio-Inspiron-5548:~/Downloads$ magick image.jpg image.png

'magic' command not found, did you mean:

   deb magic 'magic' command (8.2.157+ds.1-1)

Try it: sudo apt install <deb name>

marcio@marcio-Inspiron-5548:~/Downloads$ ./magick image.jpg image.png
bash: ./magick: Permission denied
```

Thank you in advance if you can help

**Comment by rich2005** [comment]
That is an appimage download. It is excellent. As downloaded, it is not executable and you get that error message. 
Make it executable, then as https://i.imgur.com/8Q2Ydq9.jpg

**Comment by mozgbrasil** [comment]
## https://imagemagick.org/

```bash
wget https://download.imagemagick.org/ImageMagick/download/binaries/magick

chmod +x magick

./magick --version
```
